docs: Add installation with gah (#1780)
ClementTsang/bottom: README.md

scoop install bottom
```

### gah

> On Linux and Mac OS can be installed also via [gah](https://github.com/marverix/gah):

Reviewer: ```suggestion
bottom can also be installed on Linux or macOS using gah:
```
